src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -1107,15 +1107,23 @@ export class StudyViewPageStore {
                 } : DEFAULT_LAYOUT_PROPS.dimensions[chartType]);
             });
 
+            const cancerTypeIds = _.uniq(this.queriedPhysicalStudies.result.map(study=>study.cancerTypeId));
```

Comment:
@jjgao we use the `mixed` in the cancer type id field to determine whether it is a mixed study. This should cover most of the use cases. But when user select one cancer type from a mixed study and saved as VS. Next time, when they open the VS, the survival plot should be shown by default?

Reply:
That's a good point. When opening a VS, is there a way to look at the main cancer type, if there is only one, show survival?

Alternatively, we don't have to worry about it. We will move the survival plots to comparisons tab later anyway.

Reply:
Let's postpone it.